docs/import/manage_layers.html

```diff
@@ -56,10 +56,19 @@
       </table>
     </div>
     <div id="color-fxn-editor" class="editor content-container" hidden="true">
-      <div id="color-radios"></div><br>
+      <div>
+        <input type="radio" id="SINGLE-radio" name="property-or-single"><label for="SINGLE-radio" class="radio-label">color uniformly</label>
+        <input type="radio" id="property-radio" name="property-or-single"><label for="property-radio" class="radio-label">color by property</label>
```

Comment:
I thought we discussed swapping these? That is, first selecting a property, then selecting continuous/discrete, since that seems more intuitive for a user (they're more likely to know what column they want than to say, "I really want this dataset to indicate something continuous").

If going that way, there's a nice avenue for future work to make a reasonable guess about continuous/discrete based on the column properties. Can at least have a TODO in this PR.